src/manifest.json

```diff
@@ -16,10 +16,8 @@
     "storage",
     "cookies",
     "contextMenus",
-    "https://*.simplelogin.io/*"
-  ],
-  "optional_permissions": [
     "tabs",
```

Comment:
Thank you for your proposal!
However, as far as I remember, I and @nguyenkims decided to ask for this permission **after** installation because some users don't want to enable it (maybe for privacy reason). The `tabs` permission is sensitive since there are extensions that abused this permission for tracking user's interaction, so there're users who don't want to use it (or want to use it after having tried SimpleLogin for a while).

Reply:
@ngxson for the next version, we want to simplify the onboarding process by removing this permission asking step. This is also the practice used by similar extensions like Bitwarden, 1password, Firefox Relay, etc.

As SL extension is open source, we can assure users that there's no hidden tracking.

Reply:
> As SL extension is open source, we can assure users that there's no hidden tracking.

Even when extensions are open source, it is unreasonable to just require every permission always. Most users do not have the time to review every single update. You should still be careful with what permissions you ask.